Uno.Gallery/Uno.Gallery.Shared/Views/NestedPages/MaterialTopBarSampleNestedPage.xaml

```diff
@@ -42,32 +42,32 @@
 			<Grid x:Name="Home" Visibility="Collapsed">
 				<Border Background="LightGray">
 					<StackPanel HorizontalAlignment="Center" VerticalAlignment="Center">
-						<FontIcon Glyph="{StaticResource HomeIcon}" />
-						<TextBlock Text="Home" TextAlignment="Center" />
+						<FontIcon  Foreground="{ThemeResource OnSurfaceBrush}"  Glyph="{ThemeResource HomeIcon}" />
```

Comment:
Sorry my wording was confusing in my last comment :)

The Glyphs should continue to use `StaticResource`, the Foreground should use `ThemeResource` like you have here